lib/chef/resource/kernel_module.rb

```diff
@@ -0,0 +1,100 @@
+#
+# Resource:: kernel_module
+#
+# The MIT License (MIT)
```

Comment:
This represents a new license, will need to make sure we are compliant with all the requirements. Is the new code going to also be under MIT or Apache like the rest of Chef? Does this impact our patent safety stance? (we may need to get a parent grant from Shopify).

Reply:
Lawyers have signed off on the license in this specific case